arcade/sprite_list/collision.py

```diff
@@ -93,32 +93,36 @@ def _check_for_collision(sprite1: SpriteType, sprite2: SpriteType) -> bool:
     :returns: True if sprites overlap.
     :rtype: bool
     """
+    
+    #NOTE: for speed becuase attribute look ups are slow.
```

Comment:
I surprised the linter doesn't flag this for needing a space after the #

Reply:
We need to double-check our linting settings in general. I'll add this to #1701.

Reply:
Marking this as resolved since we are evaluating our linting configuration separately.